src/extension/request_abort.c

```diff
@@ -167,6 +168,44 @@ void dd_set_response_code_and_type(int code, dd_response_type type)
     }
 }
 
+// NOLINTNEXTLINE(bugprone-easily-swappable-parameters)
+void dd_set_redirect_code_and_location(int code, zend_string *nullable location)
+{
+    _response_code = code;
+    _redirection_location = location;
+}
+
+void dd_request_abort_redirect()
+{
+    if (!_redirection_location || ZSTR_LEN(_redirection_location) == 0) {
```

Comment:
I generally don't like the use of `!<pointer>` because of the implicit cast to bool and the fact that you can't infer the type of `<pointer>`, usually I prefer `<pointer> == NULL` as it's also more explicit

Reply:
Done